app/controllers/favorites_controller.rb

```diff
@@ -3,4 +3,8 @@ class FavoritesController < ApplicationController
   def index
     @fav_pets = FavoritesList.pets
   end
+
+  def remove
+
+  end
```

Comment:
@dcoleman21 Since we didn't end up using this action let's remember to delete it in our next PR!